nanome/_internal/_plugin.py

```diff
@@ -119,6 +124,42 @@ def _on_packet_received(self, packet):
         else:
             Logs.warning("Received a packet of unknown type", packet.packet_type, ". Ignoring")
 
+    def __file_filter(self, name):
+        return name.endswith(".py")
+
+    def __file_times(self, path):
+        found_file = False
+        for file in filter(self.__file_filter, os.listdir(".")):
```

Comment:
Should this use `os.walk` instead of `listdir`? Currently `.py` files in subdirs will be ignored.

```
for root, folders, files in os.walk('.'):
    for file in filter(self.__file_filter, files):
        found_file = True
        path = os.path.join(root, file)
        yield os.stat(path).st_mtime
```